defs/go.bzl

```diff
@@ -51,7 +51,7 @@ def _compute_genrule_command(ctx):
 
 def _go_genrule_impl(ctx):
   go_toolchain = ctx.toolchains["@io_bazel_rules_go//go:toolchain"]
-  all_srcs = depset(go_toolchain.data.stdlib)
+  all_srcs = depset(go_toolchain.stdlib)
```

Comment:
hm, this still seems wrong:
```
ERROR: [path]/code/kubernetes/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/sets/BUILD:23:1: in go_genrule rule //vendor/k8s.io/apimachinery/pkg/util/sets:set-gen: 
Traceback (most recent call last):
        File "[path]/code/kubernetes/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/sets/BUILD", line 23
                go_genrule(name = 'set-gen')
        File "[path]/.cache/bazel/_bazel_jgrafton/75e5737b0f848bf1b7c3ab3303c8bf72/external/io_kubernetes_build/defs/go.bzl", line 54, in _go_genrule_impl
                depset(go_toolchain.stdlib)
cannot union value of type 'struct' to a depset
```

Reply:
looks like now we might need to use `go_toolchain.stdlib.get(ctx, go_toolchain, mode)`, though I wonder if there's a better way.

Reply:
@ixdy Correct me if I'm way off base (bazel is really new to me), but based on https://github.com/bazelbuild/rules_go/pull/956, do we even need to explicitly call in the stdlib?